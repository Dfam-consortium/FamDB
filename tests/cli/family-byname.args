family
Test family TEST0002
