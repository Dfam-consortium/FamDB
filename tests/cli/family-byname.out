>Test family TEST0003 @Other_Order [S:30,40]
GGTC
